statistics.md

```diff
@@ -32,13 +32,15 @@ Based upon the number of changes to a table, TiDB will automatically schedule [`
 
 |  System Variable | Default Value | Description |
 |---|---|---|
+| [`tidb_auto_analyze_concurrency`](/system-variables.md#tidb_auto_analyze_concurrency-new-in-v840) | `1` | The concurrency within a single automatic statistics collection task. |
```

Comment:
I think this statement is wrong. I have submitted a PR to fix it.
https://github.com/pingcap/docs/pull/20704

Reply:
@Rustin170506 I've updated the text.